.github/ISSUE_TEMPLATE/bug_report_form.yml

```
@@ -0,0 +1,64 @@
+name: Bug report
+description: Create a report to help reproduce and fix the issue.
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## **Please read the following guidelines. Follow all instructions or else your issue is subject to closure.**
+        If you are reporting an issue found in another branch or codebase, you MUST link the branch or codebase repo in your issue report or it will be closed.
+        For branches, If you have not pushed your code up, you must either reproduce it on master or push your code up before making an issue report.
+        For other codebases, if you do not have a public code repo you will be refused help unless you can completely reproduce the issue on our code.
+  - type: input
+    id: reporting-version
+    attributes:
+      label: Client Version
+      description: |
+        The BYOND version you are using to report this issue. You can find this information in the bottom left corner of the "About BYOND" window in the BYOND client.
+        It is strongly recommended that you include this, especially for concerns on the visual aspects of the game.
+      placeholder: "515.1642"
```

Also I would say using this as a placeholder has the potential to do more harm than good, since we might get thrown on wild goose-chases thinking it's not a version issue (when it secretly is, they just ignored the field)

Maybe instead something like?
```suggestion
      placeholder: "xxx.xxxx"
```
```suggestion
      placeholder: "123.4567"
```